### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_Keyboard: Keyboard focus is not Clearly Visible on the Controls(File,Edit,View,Insert,Cell...etc)present on the menubar.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3984]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10
 
**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Navigating through Keyboard on menubar.
4. Verify Keyboard focus is not visible on the Controls present on the menubar.

**Actual:**
Keyboard focus is not clearly visible on the Controls(File,Edit,View,Insert,Cell...etc)present on the menubar.

**Note:**
Same issue is observed in "**Chrome+NVDA**" Configuration.

**Expected:**
Focus should be visible through bold frame(dotted or plain) to distinguish selected and non-selected Controls(File,Edit,View,Insert,Cell...etc)present on the menubar.

**UserImpact:**
User will not be able to understand if the control is in focus or not.

**Attachment:**
(https://github.com/jupyter/notebook/files/2400350/A11y_AzureNotebooks_Jupyter_MenuBar_Keyboard.pptx)

**Comment by takluyver** [comment]
Fixed by #4679
